### ISSUE TITLE
Add description of several params in the basic usage of `torch.min()`, `torch.max()`, `torch.all()` and `torch.any()`

### ISSUE THREAD

**Issue by ILCSFNO** [pytorch/pytorch#152176]
### 📚 The doc issue

The doc of [torch.min()](https://pytorch.org/docs/stable/generated/torch.min.html#torch-min) shows its description as below three:

https://github.com/pytorch/pytorch/blob/f38dae76ee8dccd60f99bbddb48f2520f436fa1a/torch/_torch_docs.py#L7106-L7111

https://github.com/pytorch/pytorch/blob/f38dae76ee8dccd60f99bbddb48f2520f436fa1a/torch/_torch_docs.py#L7121-L7143

https://github.com/pytorch/pytorch/blob/f38dae76ee8dccd60f99bbddb48f2520f436fa1a/torch/_torch_docs.py#L7156-L7159

But for signature, it shows that:
```text
@overload
def min(input: Tensor, *, out: Optional[Tensor] = None) -> Tensor: 
@overload
def min(input: Tensor, other: Tensor, *, out: Optional[Tensor] = None) -> Tensor: 
@overload
def min(input: Tensor, dim: _int, keepdim: _bool = False, *, out: Union[Tensor, Tuple[Tensor, ...], List[Tensor], None] = None) -> torch.return_types.min: 
@overload
def min(input: Tensor, dim: Union[str, ellipsis, None], keepdim: _bool = False, *, out: Union[Tensor, Tuple[Tensor, ...], List[Tensor], None] = None) -> torch.return_types.min: 
```

So with `dim` and `other` both not specified, `out` can be used.

Tried repro below, which works well and verifies the truth:
### Repro
```python
import torch
input_data = torch.randn(3, 3)
torch.min(input_data, out=torch.tensor(0.))
```
### Output
```text
tensor(-1.6479)
```

Further, some other funcs:
* Like [torch.max()](https://pytorch.org/docs/stable/generated/torch.max.html#torch.max), [torch.all()](https://pytorch.org/docs/stable/generated/torch.all.html#torch.all) also remain these inconsistencies.
* Like [torch.any()](https://pytorch.org/docs/stable/generated/torch.any.html#torch.any) don't have any description of params in its basic usage.

So suggest to fix the basic description of `torch.min()`, `torch.max()`, `torch.all()`, `torch.any()`, and if possible, any other similar funcs.

Thanks for noting.

### Suggest a potential alternative/fix

* Suggest to fix the basic description of `torch.min()`, `torch.max()`, `torch.all()`, `torch.any()`, and if possible, any other similar funcs. 

cc @svekars @sekyondaMeta @AlannaBurke

**Comment by malfet** [comment]
Thank you for reporting. Just in general, if you spot a minor discrepancies in the doc, no need to file an issue, go ahead with the PR right away

**Comment by ILCSFNO** [comment]
Alright. I'll do it if the issue is definite. Just to escape something not sure. Thanks!

**Comment by ILCSFNO** [comment]
@malfet PR opened here:
* https://github.com/pytorch/pytorch/pull/152658
